### ISSUE TITLE
Cannot get this to work in Polarion v18.3

### ISSUE THREAD

**Issue by deckardcat** [Polarion/com.polarion.alm.extensions.codereview#18]
Cannot get this to work. Code Review section is empty; included wiki throws error "Error number 4001 in 4: Error while parsing velocity page project/AdminProject/page/Reports/Code Review
Wrapped Exception: Invocation of method 'sqlSearch' in  class $IDataService_16e5bc953df threw exception class com.polarion.platform.persistence.WrapperException : java.lang.RuntimeException: org.postgresql.util.PSQLException: ERROR: invalid input syntax for integer: "subterra:data-service:objects:/default/AdminProject${Project}AdminProject"
 Position: 198"

does this even work with v18.3?

**Comment by srcx** [comment]
Try to change 

`#set($reviewers = $trackerService.dataService.sqlSearch("SELECT USER.C_PK, COUNT(CF.C_NAME) FROM USER, WORKITEM, CF_WORKITEM CF WHERE CF.C_NAME='$codeReviewerField' AND USER.C_ID=CF.C_STRING_VALUE AND WORKITEM.C_PK=CF.FK_WORKITEM AND WORKITEM.FK_PROJECT='$project.uri' GROUP BY USER.C_PK ORDER BY COUNT(CF.C_NAME) DESC"))`

to

`#set($reviewers = $trackerService.dataService.sqlSearch("SELECT USER.C_PK, COUNT(CF.C_NAME) FROM USER, WORKITEM, CF_WORKITEM CF, PROJECT WHERE CF.C_NAME='$codeReviewerField' AND USER.C_ID=CF.C_STRING_VALUE AND WORKITEM.C_PK=CF.FK_WORKITEM AND WORKITEM.FK_PROJECT=PROJECT.C_PK AND PROJECT.C_NAME='$project.id' GROUP BY USER.C_PK ORDER BY COUNT(CF.C_NAME) DESC"))`

And please report back if it works or not, thank you.

**Comment by deckardcat** [comment]
This did the trick!  Thank you for responding.

Note that in addition to having issues with the wiki page - we can't get anything to display in the Code Review section on the work item at all (it's empty). 

We've tried to add to different work item types (User Story and Task) to no avail.  As far as we can tell, we've configured all of the required elements - even used the field ids included in the instructions.

We'll look through the logs to see if there's an error being generated.

Thanks again.
